examples/resource-method-typing/resource_method_typing.md

```diff
@@ -1,17 +1,13 @@
 # Resource method typing
 
-Resource method arguments can use user-defined types.
-Listener will use introspection to map from protocol format 
-(typically JSON) to user-defined type, using `cloneWithType`.
-Return value that is subtype of `anydata` will be mapped from 
-user-defined type to protocol format, typically JSON, using `toJson`.
-Can generate API description (e.g. OpenAPI) from Ballerina 
-service declaration.
-Annotations can be used to refine the mapping between 
-Ballerina-declared type and wire format.
+Resource method arguments can use user-defined types. Listener will use introspection to map from protocol format (typically JSON) to user-defined type, using `cloneWithType()`. Return value that is subtype of `anydata` will be mapped from user-defined type to protocol format, typically JSON, using `toJson()`. Can generate API description (e.g. OpenAPI) from Ballerina service declaration. Annotations can be used to refine the mapping between Ballerina-declared type and wire format.
```

Comment:
```suggestion
Resource method arguments can use user-defined types. The listener will use introspection to map from the protocol format (typically JSON) to the user-defined type using `cloneWithType()`. The return value, which is a subtype of `anydata` will be mapped from the user-defined type to the protocol format, typically JSON, using `toJson()`. It can generate an API description (e.g., OpenAPI) from the Ballerina service declaration. Annotations can be used to refine the mapping between the Ballerina-declared type and wire format.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/2930/commits/850aaadbbb3db21491d1ca85c58cf87a6dc589e8.